README.md

```diff
@@ -26,16 +26,39 @@ Some systems provide pre-built packages:
 
 * On Debian 9 and Ubuntu 16.04 or newer:
 
-```
-sudo apt-get install s3fs
-```
+  ```
+  sudo apt-get install s3fs
+  ```
+
+* On SUSE 12 or newer and openSUSE 42.1 or newer:
+
+  Official package:
+  ```ShellSession
+  sudo zypper in s3fs
+  ```
+
+  Or if the official package is old and you want a more recent version (not officially supported!), then check experimental versions at [https://software.opensuse.org/package/s3fs](https://software.opensuse.org/package/s3fs), and then choose one of three alternatives:
```

Comment:
The README should just give an overview; could you move the experimental notes to the wiki and link to them?  This will just leave the `zypper` command, matching other platforms.

Reply:
Sure, I was not even aware of the Wiki :-)

Reply:
Done.